packages/mobile/src/screens/chat-screen/ReactionPopup.tsx

```diff
@@ -149,23 +203,55 @@ export const ReactionPopup = ({
         {/* This 2nd pressable ensures that clicking outside of the
         message and reaction list, but inside of flatlist view,
         closes the popup. */}
-        <Pressable style={[styles.innerPressable]} onPress={handleClosePopup} />
-        <Animated.View style={{ opacity: otherOpacityAnim.current }}>
-          <ChatMessageListItem
-            chatId={chatId}
-            message={message}
-            isPopup={true}
+        <Pressable
+          style={[styles.innerPressable]}
+          onPress={() => {
+            console.log('inner press')
+            handleClosePopup()
+          }}
+        />
+        <Animated.View
+          style={{ opacity: otherOpacityAnim.current, zIndex: 40 }}
+        >
+          <View>
+            <ChatMessageListItem
+              chatId={chatId}
+              message={message}
+              isPopup={true}
+              style={[
+                styles.popupChatMessage,
+                {
+                  top: messageTop - containerTop,
+                  right: isAuthor ? spacing(6) : undefined,
+                  left: !isAuthor ? spacing(6) : undefined
+                }
+              ]}
+            />
+          </View>
+        </Animated.View>
+        <Pressable
```

Comment:
Would be worth it to pull out a `CopyButton` component here IMO, this render fn is getting pretty lengthy